core/misc/ui/jquery.ui.core.min.js

```diff
@@ -1,4 +1 @@
-/*! jQuery UI - v1.12.1 - 2017-03-31
-* http://jqueryui.com
-* Copyright jQuery Foundation and other contributors; Licensed  */
-!function(){define(["jquery","./data","./disable-selection","./focusable","./form","./ie","./keycode","./labels","./plugin","./safe-active-element","./safe-blur","./scroll-parent","./tabbable","./unique-id","./version"])}();
```

Comment:
Other compressed JS files (other than translations) seem to have kept their docblock at the top. Should this be restored/updated to match the other files?

Reply:
> Other compressed JS files (other than translations) seem to have kept their docblock at the top...

This file has no docblock in the sources, hence no comment to keep here.

See the original: https://github.com/jquery/jquery-ui/blob/1.13.0/ui/core.js
That file is now deprecated and will get removed with the 1.14 release.